docs/corsair-commander-core-guide.md

```diff
@@ -59,8 +73,9 @@ Currently, the pump and each fan can be set to a fixed duty cycle.
                channel    duty
 ```
 
-Valid channel values are `pump`, `fanN`, where 1 <= N <= 6 is the fan number, and
-`fans`, to simultaneously configure all fans.
+Valid channel values on the Core (non-XT) are `pump`, `fanN`, where 1 <= N <= 6 is the fan number.
+On the Core XT, the `pump` channel is not present and fanN starts at 0. The `fans` channel can be
```

Comment:
In liquidctl we prefer to always to number related channels starting at one, as those will be user-facing names.

Reply:
Ah, didn't realize that. I've updated the change to use 1-based indexing for user-facing names.